src/napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -557,23 +557,28 @@ def _open_grid_popup(self):
         grid_width = QtSpinBox(popup)
         grid_height = QtSpinBox(popup)
         grid_spacing = QDoubleSpinBox(popup)
+        grid_border_width = QtSpinBox(popup)
         shape_help_symbol = QtToolTipLabel(self)
         stride_help_symbol = QtToolTipLabel(self)
         spacing_help_symbol = QtToolTipLabel(self)
+        border_width_help_symbol = QtToolTipLabel(self)
 
         shape_help_msg = trans._(
             'Number of rows and columns in the grid. A value of -1 for either or both of width and height will trigger an auto calculation of the necessary grid shape to appropriately fill all the layers at the appropriate stride. 0 is not a valid entry.'
         )
 
         stride_help_msg = trans._(
-            'Number of layers to place in each grid square before moving on to the next square. The default ordering is to place the most visible layer in the top left corner of the grid. A negative stride will cause the order in which the layers are placed in the grid to be reversed. 0 is not a valid entry.'
+            'Number of layers to place in each grid quadrant before moving on to the next quadrant. The default ordering is to place the most visible layer in the top left corner of the grid. A negative stride will cause the order in which the layers are placed in the grid to be reversed. 0 is not a valid entry.'
         )
 
         spacing_help_msg = trans._(
-            'Proportional spacing between grid layers. 0 has the layers touching. Positive values will space the layers apart, and negative values will overlap the layers.'
+            'Spacing between grid quadrants. If below 1, interpreted as percentage of the quadrant size, otherwise interpreted as screen pixels. 0 has the layers touching, positive values will space the grid positions apart.'
```

Comment:
Should we use the vocab here now? not quadrants but viewboxes or viewports or whatever?